packages/core/mesh/teleport/src/muxing/rpc-port.ts

```diff
@@ -10,6 +10,6 @@ import { MaybePromise } from '@dxos/util';
  * NOTE: Copied from @dxos/rpc to avoid dependency. Structural typing should still work.
  */
 export interface RpcPort {
-  send: (msg: Uint8Array) => MaybePromise<void>;
+  send: (msg: Uint8Array, cb?: () => void) => MaybePromise<void>;
```

Comment:
What's the difference between callback and the promise here?

Reply:
i think `muxer` generally avoids `async` funcs, so it's simpler to make it work with the callbacks